cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split2/TextToSpeechSSMLGeneratorSuite.scala

```diff
@@ -0,0 +1,60 @@
+package com.microsoft.azure.synapse.ml.cognitive.split2
+
+import com.microsoft.azure.synapse.ml.cognitive._
+import com.microsoft.azure.synapse.ml.cognitive.TextToSpeechSSMLGenerator
+import com.microsoft.azure.synapse.ml.core.test.fuzzing.{TestObject, TransformerFuzzing}
+import org.apache.spark.ml.util.MLReadable
+import org.apache.spark.sql.DataFrame
+import spray.json._
+import com.microsoft.azure.synapse.ml.cognitive.TextToSpeechSSMLFormat.TextToSpeechSSMLResponseFormat
+
+class TextToSpeechSSMLGeneratorSuite extends TransformerFuzzing[TextToSpeechSSMLGenerator] with CognitiveKey  {
+
+  import spark.implicits._
+
+  def ssmlGenerator: TextToSpeechSSMLGenerator = new TextToSpeechSSMLGenerator()
+    .setUrl("https://eastus.tts.speech.microsoft.com/cognitiveservices/v1")
```

Comment:
nit: make sure this extends HasLocation and use that to make it easier for users to configure this

Reply:
Addressed in most recent commit.